homeassistant/components/mqtt/mixins.py

```diff
@@ -414,11 +411,6 @@ async def async_will_remove_from_hass(self) -> None:
             self.hass, self._attributes_sub_state
         )
 
-    @property
-    def extra_state_attributes(self) -> dict[str, Any] | None:
-        """Return the state attributes."""
-        return self._attributes
-
```

Comment:
Let's stop right here and await any further refactoring and actions until the Event entity has finalized. 

You are getting way ahead of things at this point.

Reply:
I understand this from the perspective of an MqttEventEntity or EventEntity, but this refactoring has no direct link to that.
I did not mean to rush here.
I believe it is okay to directly set `_attr_extra_state_attributes` instead of implementing `extra_state_attributes`, which is still done for MQTT Siren BTW. So I would open this PR even when the Event entity would not be realized or in a different form.